build.Dockerfile

```diff
@@ -116,14 +129,20 @@ COPY --from=builder /home/ubuntu/work/worker/bin/* ./
 COPY --from=builder /lib/x86_64-linux-gnu/libsgx* /lib/x86_64-linux-gnu/
 COPY --from=builder /lib/x86_64-linux-gnu/libdcap* /lib/x86_64-linux-gnu/
 
-RUN touch spid.txt key.txt
+# RUN touch spid.txt key.txt
```

Comment:
We can remove this even in the IAS case as these files are no longer mandatory if we run in software mode and skip the remote attestation.